types/quill/index.d.ts

```diff
@@ -84,9 +84,9 @@ export interface DeltaStatic {
     concat(other: DeltaStatic): DeltaStatic;
     diff(other: DeltaStatic, index?: number): DeltaStatic;
     eachLine(predicate: (line: DeltaStatic, attributes: StringMap, idx: number) => any, newline?: string): DeltaStatic;
-    transform(index: number): number;
+    transform(index: number, priority?: boolean): number;
     transform(other: DeltaStatic, priority: boolean): DeltaStatic;
-    transformPosition(index: number): number;
+    transformPosition(index: number, priority?: boolean): number;
```

Comment:
According to this [doc](https://github.com/quilljs/delta/#transformposition), priority is only on transform(other: DeltaStatic, priority: boolean), not on `transformPosition(index)`

Reply:
updating docs as well: https://github.com/quilljs/delta/pull/21